libkbfs/keybase_service_base.go

```diff
@@ -490,6 +490,7 @@ func (k *KeybaseServiceBase) Identify(ctx context.Context, assertion, reason str
 		k.log.CDebugf(ctx,
 			"Ignoring error (%s) for user %s with no sigchain; "+
 				"error type=%T", err, res.Ul.Name, err)
+		ei.onError(ctx)
```

Comment:
Looks like we're not actually ignoring the error here anymore, so that debug line is incorrect?

Reply:
We're still ignoring the error in the sense that we're not returning it from this function.  The `onError` call is just making sure that some cleanup code runs (in place of the `userBreak` or `teamBreak` calls below, which won't get run now).